src/others/file_explorer/lv_file_explorer_private.h

```diff
@@ -51,6 +51,18 @@ struct _lv_file_explorer_t {
     lv_file_explorer_sort_t sort;
 };
 
+typedef enum {
+    FILE_KIND_DIR,
+    FILE_KIND_IMAGE,
+    FILE_KIND_AUDIO,
+    FILE_KIND_VIDEO,
+    FILE_KIND_FILE
```

Comment:
These should be namespaced a bit more.

```suggestion
    LV_FILE_EXPLORER_FILE_KIND_DIR,
    LV_FILE_EXPLORER_FILE_KIND_IMAGE,
    LV_FILE_EXPLORER_FILE_KIND_AUDIO,
    LV_FILE_EXPLORER_FILE_KIND_VIDEO,
    LV_FILE_EXPLORER_FILE_KIND_FILE
```